BossMod/Modules/Endwalker/Trials/T02Hydaelyn/Echoes.cs

```diff
@@ -0,0 +1,26 @@
+namespace BossMod.Endwalker.Trials.T02Hydaelyn
+{
+    class Echoes : Components.UniformStackSpread
+    {
+        public int NumCasts { get; private set; }
+
+        public Echoes() : base(6, 0, 8) { }
+
+        public override void OnEventCast(BossModule module, Actor caster, ActorCastEvent spell)
+        {
+            if ((AID)spell.Action.ID == AID.Echoes)
+                ++NumCasts;
+                if (NumCasts == 5)
```

Comment:
This is extremely misleading. It is indented, as if it were part of condition above, but in reality it is not.

Please use auto formatting.

Reply:
it was working as intended, but sure i can add another set of brackets to it